src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -139,4 +139,33 @@ public int getTotalMessageCount() {
     PreparedQuery results = datastore.prepare(new Query("Message"));
     return results.countEntities(FetchOptions.Builder.withLimit(1000));
   }
+
+  //Stores the user data
```

Comment:
When adding a description to a function it usually uses the following format:
/** <Comment> */

https://google.github.io/styleguide/javaguide.html#s7.1.1-javadoc-multi-line

Reply:
All fixed! I think these two functions are actually from Nikita– I resolved a merge conflict earlier so I have the About Me functions in here as well now.